packages/fxa-payments-server/src/App.tsx

```diff
@@ -180,7 +180,12 @@ export const AppErrorDialog = ({ error: { message } }: { error: Error }) => {
   // Most of these errors will probably be failure to load Stripe widgets.
   return (
     <SettingsLayout>
-      <DialogMessage className="dialog-error" onDismiss={locationReload}>
+      <DialogMessage
+        className="dialog-error"
+        onDismiss={locationReload}
+        headerId="general-application-error-header"
```

Comment:
You'll need to assign these new IDs to the headers/content. See here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role